rails-tasks/app/controllers/tasks_controller.rb

```diff
@@ -29,11 +31,33 @@ def create
   end
 
   def destroy
-    to_delete = Task.where(task_params[:name])
-    Task.destroy(to_delete[0].id)
+    to_delete = Task.find_by(name: params[:name])
```

Comment:
I like this use of `find_by` over `where`, because we know we're only ever going to remove a single task in this route.